Makefile

```diff
@@ -45,11 +45,17 @@ ci-non-test: verify licenses vulncheck ## Runs checks other than tests
 ci: toolchain ci-non-tests ci-tests ## Run all steps used by continuous integration
 
 run: ## Run Karpenter controller binary against your local cluster
-	SYSTEM_NAMESPACE=${SYSTEM_NAMESPACE} go run ./cmd/controller/main.go \
-		--cluster-name=${CLUSTER_NAME} \
-		--cluster-endpoint=${CLUSTER_ENDPOINT} \
-		--aws-default-instance-profile=KarpenterNodeInstanceProfile-${CLUSTER_NAME} \
-		--leader-elect=false
+	kubectl create configmap -n ${SYSTEM_NAMESPACE} karpenter-global-settings \
```

Comment:
This command isn't idempotent and will fail if run twice. Can we point to a local file?

Reply:
It is idempotent, see the `kubectl apply` with the `--dry-run`